datafusion/core/src/dataframe.rs

```diff
@@ -344,24 +344,30 @@ impl DataFrame {
             .collect::<Vec<_>>();
         describe_schemas.insert(0, Field::new("describe", DataType::Utf8, false));
 
+        //count aggregation
+        let cnt = self.clone().count().await?;
+
         //collect recordBatch
         let describe_record_batch = vec![
             // count aggregation
-            self.clone()
-                .aggregate(
-                    vec![],
-                    fields_iter
+            vec![RecordBatch::try_new(
```

Comment:
I don't fully understand this change, but it seems ok.

Reply:
This doesn't fix the issue [5444](https://github.com/apache/arrow-datafusion/issues/5444), but it did lead me to discover a better way to speed up the operation of describe.